src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -163,5 +166,22 @@ public int MaxModelValidationErrors
         /// </ul>
         /// </remarks>
         public IList<IMetadataDetailsProvider> ModelMetadataDetailsProviders { get; }
+
+        /// <summary>
+        /// Provides programmatic configuration for the html helpers and view context.
```

Comment:
- `AntiForgeryOptions` is the only property in this class using "Provides". Better to stick with "Gets" e.g. "Gets the configuration information for ..."
- "the html" -> HTML
- "view context" -> `<see cref="ViewContext"/>`